NOTICE

```diff
@@ -1 +1 @@
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Looks like it happening since ACK rt 0.14.0 - rds-controller got the same change https://github.com/aws-controllers-k8s/rds-controller/blob/main/NOTICE

Reply:
yeah, I've seen it happen a few times.... always wondered what was going on there.

Reply:
In the API schema description `LayerList` is a list of strings and not `[]*Layer`. I will address this issue in a future PR